packages/@react-spectrum/provider/src/Provider.tsx

```diff
@@ -149,7 +149,8 @@ const ProviderWrapper = React.forwardRef(function ProviderWrapper(props: Provide
     ...styleProps.style,
     // This ensures that browser native UI like scrollbars are rendered in the right color scheme.
     // See https://web.dev/color-scheme/.
-    colorScheme: props.colorScheme ?? Object.keys(theme).filter(k => k === 'light' || k === 'dark').join(' ')
+    colorScheme: props.colorScheme ?? Object.keys(theme).filter(k => k === 'light' || k === 'dark').join(' '),
+    opacity: .99
```

Comment:
Will probably be changing this to .9999 in a followup PR (along with changing the Modal opacity to .9999 so that the background docs aren't just _barely visible_ behind the modal)

Reply:
Can you add this to the `Overlay` component rather than all providers? And maybe use the `isolation` property instead of the opacity hack?

Reply:
sure lemme give that a shot